src/main/java/com/eternalcode/discordapp/DiscordApp.java

```diff
@@ -55,7 +55,7 @@ public class DiscordApp {
     private static ExperienceRepository experienceRepository;
 
     public static void main(String... args) throws InterruptedException {
-        YamlFilesManager configManager = new YamlFilesManager("config");
+        ConfigManager configManager = new ConfigManager("config");
```

Comment:
Revert nazwę klasy, configManager służy jak nazwa mówi od konfiguracji stricte, a wykorzystywane jest w m.in experience system jako bot data (voice exp)

Reply:
To, że coś przechowuje informacje dotyczące bazy danych w pliku, nie oznacza, że trzeba zmieniać nazwę klasy i tworzyć osobny pakiet "data", który przechowuje tylko jedną klasę.

Reply:
Ale nazwa `ConfigManager` stricte mówi że niby służy tylko do konfiguracji - według mnie jak było po moim PR było dobrze

Reply:
Nic to nie zmienia

Reply:
W każdym projekcie korzystamy z tej nazwy. Nikt nigdy nie wpadł na pomysł, żeby nagle pójść do np. EternalCore tylko po to, żeby zmienić nazwę pliku i przenieść ją do osobnej paczki z powodu przechowywania kilku warpów w pliku `locations.yml`

To że przechowywujesz tymczasowo informacje dot. exp nie oznacza że nagle to coś nie staję się konfiguracją, przechowywujesz w niej po prostu dane takie jakbyś mógł przechowywać dane w bazie danych.

## Moje uzasadnienie w tej sprawie:
1. W naszym projekcie nazwa ConfigManager była używana przez pewien czas i była szeroko znana. Nie widzę potrzeby zmieniać nazwę tylko ze względu na przechowywanie kilku danych tymczasowych.
2. przechowywanie danych tymczasowych nie wyklucza możliwości, że te dane są również częścią konfiguracji. 
3. **Ważne jest utrzymanie spójności w naszym projekcie. Jeśli zmienimy nazwę tylko dla jednego pliku lub funkcjonalności, może to prowadzić do dezorganizacji i zamieszania w kodzie.**
4. Nie ma potrzeby tworzenia osobnego pakietu "data", który przechowuje tylko `jedną klasę`, gdyż przechowywanie informacji dotyczących bazy danych w pliku nie wymaga zmiany nazwy klasy ani tworzenia nowego pakietu.

Reply:
Nazwa klasy nie powinna być "jedyna i słuszna", wszystko to mocno **zależy** od kontekstu, powtarzam **zależy**. I akurat w tym kontekście ta nazwa klasy prawidłowo opisuje za co faktycznie odpowiada.

Reply:
> W każdym projekcie korzystamy z tej nazwy. Nikt nigdy nie wpadł na pomysł, żeby nagle pójść do np. EternalCore tylko po to, żeby zmienić nazwę pliku i przenieść ją do osobnej paczki z powodu przechowywania kilku warpów w pliku `locations.yml`
> 
> To że przechowywujesz tymczasowo informacje dot. exp nie oznacza że nagle to coś nie staję się konfiguracją, przechowywujesz w niej po prostu dane takie jakbyś mógł przechowywać dane w bazie danych.
> 
> ## Moje uzasadnienie w tej sprawie:
> 1. W naszym projekcie nazwa ConfigManager była używana przez pewien czas i była szeroko znana. Nie widzę potrzeby zmieniać nazwę tylko ze względu na przechowywanie kilku danych tymczasowych.
> 2. przechowywanie danych tymczasowych nie wyklucza możliwości, że te dane są również częścią konfiguracji. 
> 3. **Ważne jest utrzymanie spójności w naszym projekcie. Jeśli zmienimy nazwę tylko dla jednego pliku lub funkcjonalności, może to prowadzić do dezorganizacji i zamieszania w kodzie.** **(doświadczyłem tego podczas wykonywania PR GH-92)**
> 4. Nie ma potrzeby tworzenia osobnego pakietu "data", który przechowuje tylko `jedną klasę`, gdyż przechowywanie informacji dotyczących bazy danych w pliku nie wymaga zmiany nazwy klasy ani tworzenia nowego pakietu.

To ChatGPT wygenerował? xD

Reply:
> > W każdym projekcie korzystamy z tej nazwy. Nikt nigdy nie wpadł na pomysł, żeby nagle pójść do np. EternalCore tylko po to, żeby zmienić nazwę pliku i przenieść ją do osobnej paczki z powodu przechowywania kilku warpów w pliku `locations.yml`
> > 
> > To że przechowywujesz tymczasowo informacje dot. exp nie oznacza że nagle to coś nie staję się konfiguracją, przechowywujesz w niej po prostu dane takie jakbyś mógł przechowywać dane w bazie danych.
> > 
> > ## Moje uzasadnienie w tej sprawie:
> > 1. W naszym projekcie nazwa ConfigManager była używana przez pewien czas i była szeroko znana. Nie widzę potrzeby zmieniać nazwę tylko ze względu na przechowywanie kilku danych tymczasowych.
> > 2. przechowywanie danych tymczasowych nie wyklucza możliwości, że te dane są również częścią konfiguracji. 
> > 3. **Ważne jest utrzymanie spójności w naszym projekcie. Jeśli zmienimy nazwę tylko dla jednego pliku lub funkcjonalności, może to prowadzić do dezorganizacji i zamieszania w kodzie.** **(doświadczyłem tego podczas wykonywania PR GH-92)**
> > 4. Nie ma potrzeby tworzenia osobnego pakietu "data", który przechowuje tylko `jedną klasę`, gdyż przechowywanie informacji dotyczących bazy danych w pliku nie wymaga zmiany nazwy klasy ani tworzenia nowego pakietu.
> 
> To ChatGPT wygenerował? xD

Ty lepiej zrób review a nie wypowiadaj się w takim sensie. Nie o to cie prosiłem

Reply:
> > Nazwa klasy nie powinna być "jedyna i słuszna", wszystko to mocno **zależy** od kontekstu, powtarzam **zależy**. I akurat w tym kontekście ta nazwa klasy prawidłowo opisuje za co faktycznie odpowiada.
> 
> Czyli która?

YamlFilesManager 

Lecz mi osobiście bardziej tu pasuje coś typu YamlFilesStorage

Reply:
> > To ChatGPT wygenerował? xD
> 
> Ty lepiej zrób review a nie wypowiadaj się w takim sensie. Nie o to cie prosiłem 
> 

Widzę, że dyskusja o nazwę klasy na poziomie. Tobie nie zależy na dojściu do dobrego rozwiązania tylko nad udowodnieniem swojej racji. A i tak to w finale nie ma to znaczenia.